lib/rules/max-empty-lines/index.js

```diff
@@ -67,18 +64,19 @@ function rule(max, options, context) {
 			const rootRawsAfter = root.raws.after;
 
 			// not document node
+			// @ts-expect-error -- TS2339: Property 'document' does not exist on type 'Root'.
```

Comment:
We'll likely need to revisit this `document` stuff at some point in light of https://github.com/postcss/postcss/pull/1582.

I think it's mainly the max whitespace rules that touch it.

Reply:
Yes, I also think so. 👍🏼 
I didn't know how I should handle it, so I've left `// @ts-expect-error` comments.

Reply:
Leaving `// @ts-expect-error` comments was the right thing to do.

Feel free to merge.

Reply:
Thank you!